include/mgp.hpp

```
@@ -3480,6 +3482,54 @@ inline bool Value::operator<(const Value &other) const {
   }
 }
 
+inline std::ostream &operator<<(std::ostream &os, const mgp::Value &value) {
+  switch (value.Type()) {
+    case mgp::Type::Null:
+      return os << "null";
+    case mgp::Type::Any:
+      return os << "any";
+    case mgp::Type::Bool:
+      return os << (value.ValueBool() ? "true" : "false");
+    case mgp::Type::Int:
+      return os << std::to_string(value.ValueInt());
+    case mgp::Type::Double:
+      return os << std::to_string(value.ValueDouble());
+    case mgp::Type::String:
+      return os << std::string(value.ValueString());
+    case mgp::Type::List:
+      return os << "List[]";
+    case mgp::Type::Map:
+      return os << "Map[]";
```

Do we want to throw an exception maybe for this? As it is not implemented?